internal/commands/test.go

```diff
@@ -159,6 +164,8 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 	cmd.Flags().Bool("no-color", false, "Disable color when printing")
 	cmd.Flags().Bool("suppress-exceptions", false, "Do not include exceptions in output")
 	cmd.Flags().Bool("all-namespaces", false, "Test policies found in all namespaces")
+	cmd.Flags().Bool("quiet", false, "Disable successful test output")
+	cmd.Flags().Bool("suppress-no-policies-found", true, "Suppresses output that no policies found")
```

Comment:
I think we can default this to false, so we could use the flag to activate the behavior 
```shell
./conftest test -p policy/empty.rego deployment.yaml --suppress-no-policies-found
```